#! /bin/sh /usr/share/dpatch/dpatch-run
## 01-info-section.dpatch by Nikita V. Youschenko <yoush@debian.org>
##
## DP: Changes @dircategory in .texi files, to fit better into debian
## DP: info directory.

@DPATCH@
diff -urNad gcc-4.1-doc-non-dfsg-4.1.1~/fortran-texi/gfortran.texi gcc-4.1-doc-non-dfsg-4.1.1/fortran-texi/gfortran.texi
--- gcc-4.1-doc-non-dfsg-4.1.1~/fortran-texi/gfortran.texi	2006-03-05 21:34:17.000000000 +0300
+++ gcc-4.1-doc-non-dfsg-4.1.1/fortran-texi/gfortran.texi	2006-09-17 23:43:25.527155834 +0400
@@ -61,7 +61,7 @@
 @end copying
 
 @ifinfo
-@dircategory Programming
+@dircategory Development
 @direntry
 * gfortran: (gfortran).                  The GNU Fortran 95 Compiler.
 @end direntry
diff -urNad gcc-4.1-doc-non-dfsg-4.1.1~/texi/cpp.texi gcc-4.1-doc-non-dfsg-4.1.1/texi/cpp.texi
--- gcc-4.1-doc-non-dfsg-4.1.1~/texi/cpp.texi	2005-09-21 00:31:37.000000000 +0400
+++ gcc-4.1-doc-non-dfsg-4.1.1/texi/cpp.texi	2006-09-17 23:43:00.343484968 +0400
@@ -50,7 +50,7 @@
 @set cppmanual
 
 @ifinfo
-@dircategory Programming
+@dircategory Development
 @direntry
 * Cpp: (cpp).		       The GNU C preprocessor.
 @end direntry
diff -urNad gcc-4.1-doc-non-dfsg-4.1.1~/texi/cppinternals.texi gcc-4.1-doc-non-dfsg-4.1.1/texi/cppinternals.texi
--- gcc-4.1-doc-non-dfsg-4.1.1~/texi/cppinternals.texi	2005-03-05 22:56:31.000000000 +0300
+++ gcc-4.1-doc-non-dfsg-4.1.1/texi/cppinternals.texi	2006-09-17 23:43:08.749372415 +0400
@@ -5,7 +5,7 @@
 @include gcc-common.texi
 
 @ifinfo
-@dircategory Programming
+@dircategory Development
 @direntry
 * Cpplib: (cppinternals).      Cpplib internals.
 @end direntry
diff -urNad gcc-4.1-doc-non-dfsg-4.1.1~/texi/gcc.texi gcc-4.1-doc-non-dfsg-4.1.1/texi/gcc.texi
--- gcc-4.1-doc-non-dfsg-4.1.1~/texi/gcc.texi	2005-06-29 04:14:32.000000000 +0400
+++ gcc-4.1-doc-non-dfsg-4.1.1/texi/gcc.texi	2006-09-17 23:43:04.973321406 +0400
@@ -62,7 +62,7 @@
      funds for GNU development.
 @end copying
 @ifnottex
-@dircategory Programming
+@dircategory Development
 @direntry
 * gcc: (gcc).                  The GNU Compiler Collection.
 @end direntry
diff -urNad gcc-4.1-doc-non-dfsg-4.1.1~/texi/gccint.texi gcc-4.1-doc-non-dfsg-4.1.1/texi/gccint.texi
--- gcc-4.1-doc-non-dfsg-4.1.1~/texi/gccint.texi	2005-03-14 23:18:43.000000000 +0300
+++ gcc-4.1-doc-non-dfsg-4.1.1/texi/gccint.texi	2006-09-17 23:43:13.639143524 +0400
@@ -48,7 +48,7 @@
      funds for GNU development.
 @end copying
 @ifnottex
-@dircategory Programming
+@dircategory Development
 @direntry
 * gccint: (gccint).            Internals of the GNU Compiler Collection.
 @end direntry
diff -urNad gcc-4.1-doc-non-dfsg-4.1.1~/texi/treelang.texi gcc-4.1-doc-non-dfsg-4.1.1/texi/treelang.texi
--- gcc-4.1-doc-non-dfsg-4.1.1~/texi/treelang.texi	2005-06-25 04:59:41.000000000 +0400
+++ gcc-4.1-doc-non-dfsg-4.1.1/texi/treelang.texi	2006-09-17 23:42:53.060314249 +0400
@@ -100,7 +100,7 @@
 @end copying
 
 @ifnottex
-@dircategory Programming
+@dircategory Development
 @direntry
 * treelang: (treelang).                  The GNU Treelang compiler.
 @end direntry
